### PR TITLE
Awesome books: refactor to use JavaScript classes

### PR DESCRIPTION
**Pull request by fbclh**
**Project Requirements:** 

- Use JavaScript classes instead of objects and arrays.
- Create class methods to add and remove books.
- Maintain all of the original functionality of the application 
- Don't use any library or framework, use only plain JavaScript.
- Add CSS styles to the application to make it match this wireframe:
- You can use the styles of your choice, but try to follow the general layout.
- Concentrate on the general layout, rather than the details.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+/* eslint-disable max-classes-per-file */
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.id = new Date().valueOf().toString();
+  }
+}
+
+const removeElement = (element) => {
```
Comment (mateo951) [inline index.js:10]: Awesome use of arrow functions 👌

File: index.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
+/* eslint-disable max-classes-per-file */
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.id = new Date().valueOf().toString();
+  }
+}
```
Comment (mateo951) [inline index.js:8]: Fantastic 🙌 Proper use of JavaScript classes instead of objects and arrays as expected!

File: index.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  }
+
+  addBook(book) {
+    this.list.push(book);
+  }
+
+  updateDom() {
+    const list = document.querySelector('.listing');
+    removeElement(list);
+    this.list.forEach((el) => {
```
Comment (mateo951) [inline index.js:33]: As you can see in the following image there is a tiny issue that is preventing the user to add a new book 😥

image

Please fix this issue so your project meets the project requirements 📝 Your app should be able to add and remove books 🤓 Kindly make sure the refactoring did not introduce bugs to the app in other areas as well!

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi Team, 

📝👌 **Your project highlights** 🔥😎 
- General Requirements ✔
- JS Best Practices ✔

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Review by od-c0d3r — APPROVED** [review]
Asslam Alaikum Team @fbclh @franklinben23 ,😊

Hi👋🏽, Your project is complete!  There is nothing else to say other than... it's time to merge it 😎 

# Congratulations! 🎉 

Happy coding!👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**